exercise3/index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+spl_autoload_register(function ($class) {
+    $prefix = 'exercise3\\';
+    $baseDir = __DIR__ . '/src/';
+    $relativeClass = str_replace($prefix, '', $class);
+    $file = $baseDir . str_replace('\\', '/', $relativeClass) . '.php';
+
+    if (file_exists($file)) {
+        require $file;
+    }
+});
+
+use exercise3\Controller\FibonacciController;
+use exercise3\Model\FibonacciModel;
+use exercise3\Views\FibonacciView;
```

Comment:
I think this is better to write this on top of the file for readability and maintainability.